sql/updates/world/3.3.5/2016_09_32_17853_world_335.sql

```diff
@@ -0,0 +1,48 @@
+-- NPC ID 4489 Braug Dimspirit, Test of Lore
+SET @Braug  := 4489;
+SET @menu   := 4763;
+SET @quest  := 6627;
+SET @Script := 448900;
+
+UPDATE `creature_template` SET `AIName`= 'SmartAI', `ScriptName`= '' WHERE `entry`= @Braug;
+
+DELETE FROM `creature_text` WHERE `entry`= @Braug;
+INSERT INTO `creature_text` (`entry`,`groupid`,`id`,`text`,`type`,`language`,`probability`,`emote`,`duration`,`sound`,`BroadcastTextId`,`TextRange`,`comment`) VALUES
+(@Braug, 0,0,'Ah, Ysera the Dreamer.',                                12,0,100,1,3000,0,8382,0, 'Braug Dimspirit - Test of Lore'),
+(@Braug, 1,0,'Neltharion, hmm? As good an answer as any, yes?',       12,0,100,1,3000,0,8386,0, 'Braug Dimspirit - Test of Lore'),
+(@Braug, 2,0,'Alexstrazsa, the Life-Binder. A magnificent creature.', 12,0,100,1,3000,0,8387,0, 'Braug Dimspirit - Test of Lore'),
+(@Braug, 3,0,'Malygos, is it?',                                       12,0,100,1,3000,0,8388,0, 'Braug Dimspirit - Test of Lore'),
+(@Braug, 4,0,'That is the wrong answer, $n. Be off with you.',        12,0,100,1,3000,0,8393,0, 'Braug Dimspirit - Test of Lore');
+
+DELETE FROM `gossip_menu_option` WHERE `menu_id` = @menu;
+INSERT INTO `gossip_menu_option` (`menu_id`,`id`,`option_icon`,`option_text`,`OptionBroadcastTextID`,`option_id`,`npc_option_npcflag`,`action_menu_id`,`action_poi_id`,`box_coded`,`box_money`,`box_text`,`BoxBroadcastTextID`,`VerifiedBuild`) VALUES
+(@menu, 0, 0, 'Ysera is my answer.',        8381, 1, 1, 0, 0, 0, 0, '', 0, 0),
+(@menu, 1, 0, 'Neltharion is  my answer.',  8383, 1, 1, 0, 0, 0, 0, '', 0, 0),
+(@menu, 2, 0, 'Alexstrasza is  my answer.', 8384, 1, 1, 0, 0, 0, 0, '', 0, 0),
```

Comment:
Yes, that is a correct observation. :-) But that is "unfortunately" also what is in `broadcast_text` :
```sql
SELECT * FROM `broadcast_text` WHERE `ID` IN (8381,8383,8384,8385);
```
```
  ID  Language  MaleText                    FemaleText                  EmoteID0  EmoteID1  EmoteID2  EmoteDelay0  EmoteDelay1  EmoteDelay2  SoundId    Unk1    Unk2  VerifiedBuild
----  --------  --------------------------  --------------------------  --------  --------  --------  -----------  -----------  -----------  -------  ------  ------  -------------
8381         0  Ysera is my answer.         Ysera is my answer.                0         0         0            0            0            0        0       0       1          18019
8383         0  Neltharion is  my answer.   Neltharion is  my answer.          0         0         0            0            0            0        0       0       1          18019
8384         0  Alexstrasza is  my answer.  Alexstrasza is  my answer.         0         0         0            0            0            0        0       0       1          18019
8385         0  Malygos is my answer.       Malygos is my answer.              0         0         0            0            0            0        0       0       1          18019
```
As you probably know, most/all text rows with reference to `broadcast_text` will use the `broadcast_text` content. 
If you think it is worth while changing the content in `broadcast_text`.`ID` 8383+8384, I will gladly change it here. ;-)